src/main/java/org/apache/commons/csv/CSVRecord.java

```diff
@@ -144,6 +159,15 @@ public long getCharacterPosition() {
         return characterPosition;
     }
 
+    /**
+     * Returns the start byte of this record as a character byte in the source stream.
+     *
+     * @return the start byte of this record as a character byte in the source stream.
```

Comment:
Either the code or the comment is wrong. The return value is a `long` but the comment talks about a `byte`. It would help to make the comment clear as to why the return type is a `long` if that is indeed correct.